### DISCUSSION TITLE
Training time

### DISCUSSION THREAD

**Discussion by us8329** [MIC-DKFZ/nnUNet#1147] [Q&A]
Hi , Im using a ubuntu ssh with high performance gpu to train my nnU-net model for emphysema segmentation on jupyter notebook. Each epoch in nnUNetTrainerV2 is taking 3 minutes and there are 1000 epochs.Can someone suggest a way to complete the training , as I keep 'losing connection' from the notebook or get a 'broken pipe' from my ssh. 
I'm working on MacOs and have tried tmux , clicker apps to keep the notebook from going inactive.

**Comment by MathijsdeBoer** [comment]
Are you running TMUX on your local machine, or in your SSH environment?
What I usually do is `ssh user@server`, provide login information and if I haven't got a session running, simply type `tmux`.
If I already have an earlier session, simply typing `tmux attach` will return you to the last active session.
